Dockerfile

```diff
@@ -3,6 +3,8 @@ FROM geoffreybooth/meteor-base:2.5 AS meteor-builder
 
 ARG APPNAME
 
+RUN apt update && apt install -y ssh-client
+
```

Comment:
Lets remove these two lines - they're no longer needed now that we're using jspdf 2.5.

Reply:
Done!